package.json

```diff
@@ -93,6 +93,16 @@
                 "when": "editorTextFocus && vim.useCtrlKeys"
             },
             {
+                "key": "ctrl+a",
```

Comment:
I wasn't quite sure what that was for, since a lot (some?) of the other ones don't have that guard.